src/Illuminate/Routing/Route.php

```diff
@@ -722,6 +732,29 @@ public function middleware($middleware = null)
     }
 
     /**
+     * Remove the given middlewares from the route. If no middleware is passed, all middlewares will be removed.
+     *
+     * @param null $middleware
+     * @return $this
+     */
+    public function withoutMiddleware($middleware = null)
+    {
+        if (is_null($middleware)) {
```

Comment:
I kept the `is_null` because the `middleware` method uses it and I was going for consistency. Do you think I should use $middlware === null anyway?